### PR TITLE
Add formula for input-source-switcher

### PR DESCRIPTION
**Pull request by tarrant**
This provides a command line method for switching between multiple input
sources.

### CODE REVIEW

File: Library/Formula/input-source-switcher.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class InputSourceSwitcher < Formula
+  homepage "https://github.com/vovkasm/input-source-switcher"
+  url "https://github.com/vovkasm/input-source-switcher/archive/cfca8149c6805b333910858a22aeeb124cbca903.zip"
```
Comment (mistydemeo) [inline Library/Formula/input-source-switcher.rb]: Is there a stable release for this software? We only accept software with stable releases.

Comment (tarrant) [inline Library/Formula/input-source-switcher.rb]: There are no tagged releases, but the code hasn't changed in six months so I'd consider that stable. Is the only way to get this accepted to find the original author and get them to tag it?

Comment (mistydemeo) [inline Library/Formula/input-source-switcher.rb]: Yes, please!

**Comment by tarrant** [comment]
Ok, switched to a tagged release.

**Comment by mistydemeo** [comment]
Thanks!

`brew audit` flags a few things:

```

==> audit problems
input-source-switcher:
 * Stable: version 0.2 is redundant with version scanned from URL
 * Use GitHub tarballs rather than zipballs (url is https://github.com/vovkasm/input-source-switcher/archive/v0.2.zip).
 * Use `system "#{bin}/issw", "-l"` instead of `system "#{bin}/issw -l"` 
```

**Comment by tarrant** [comment]
I've fixed the thing that audit complained about.

**Comment by DomT4** [comment]
@Homebrew/owners Do we want to carry this? Only one publicly released version, 21 commits, only 1 Github star. Dev seems active, but don't know if it's too new for us.

**Comment by mikemcquaid** [comment]
Yeh, this is a little niche for us as-is I'm afraid. Sorry!

**Pull request closed without merging**